src/Psalm/Internal/PropertyMap.php

```diff
@@ -256,6 +248,62 @@
     'mongoint64' => [
         'value' => 'string',
     ],
+    'mysqli' => [
+        'affected_rows' => 'int',
+        'client_info' => 'string',
+        'client_version' => 'int',
+        'connect_errno' => 'int',
+        'connect_error' => 'string',
+        'errno' => 'int',
+        'error' => 'string',
+        'error_list' => 'array',
+        'field_count' => 'int',
+        'host_info' => 'string',
+        'info' => 'string',
+        'insert_id' => 'mixed',
+        'protocol_version' => 'string',
+        'server_info' => 'string',
+        'server_version' => 'int',
+        'sqlstate' => 'string',
+        'stat' => 'mixed',
```

Comment:
the phpdocs do not define a stat property...? Should it be dropped ?

Reply:
Yeah, looks like it's actually a method: https://www.php.net/manual/en/mysqli.stat.php

Reply:
https://github.com/mpesari/psalm/pull/1